src/core/js/models/itemsQuestionModel.js

```diff
@@ -30,6 +30,8 @@ define([
     init() {
       super.init();
       this.set('_isRadio', this.isSingleSelect());
+      this.listenTo(this.getChildren(), 'change:_isActive': this.checkCanSubmit);
```

Comment:
This line throws an error when trying to you try to build the course.

```
    Error: Line 33: Unexpected token :
    In module tree:
        core/js/app
          components/adapt-contrib-gmcq/js/adapt-contrib-gmcq
            components/adapt-contrib-mcq/js/adapt-contrib-mcq
```

Reply:
Apologies, try now.